CHANGES.md

```diff
@@ -14,6 +14,8 @@
 
 - Install `primer.json` (used by `black-primer` by default) with black. (#2154)
 
+- Add passing `posargs` to `pytest` in `tox.ini`. (#2173)
```

Comment:
I don't think this change log entry is necessary since this is only for Black contributors. The change log for user-facing changes. I'll give your PR a `skip news` label instead.